tests/e2e/state.go

```diff
@@ -38,7 +39,7 @@ type State map[ChainID]ChainState
 
 type Chain struct {
 	target     e2e.TargetDriver
-	testConfig TestConfig
+	testConfig *TestConfig
```

Comment:
Why did we perform this change?

Reply:
This was due to the spawn time where Now is used as time reference. if you update this on 'actions' it's out of sync from 'state'. so I kept this change even if not needed anymore as we don't need to do it for permissionless test.